ansible/templates/zapd_balance.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+
+bal=`curl -s -d '{"jsonrpc":"2.0","id":1,"method":"getbalance","params":{}}' -H "Content-Type: application/json-rpc" localhost:5000/api | jq '.["result"]["balance"]'`
+min_bal=500000
```

Comment:
lets make constants min_bal 5000 and max_bal 10000, then when you use them here multiply by  a constant CENTS (which will be 100 obviously), this way you can reuse the min/max constants in the email message